### PR TITLE
added delete review function in controllr

### PR DESCRIPTION
**Pull request by Jyaheza**

### CODE REVIEW

File: app/controllers/review.controller.js [hunk 1]
```diff
@@ -44,11 +44,39 @@ exports.create = async (req, res) => {
...
+
+  exports.delete = async (req, res) => {
+    const reviewId = req.params.id;
+    try {
+        const result = await ResumeReview.destroy({
+            where: { id: reviewId }
+        });
+
+        if (result === 0) {
+            console.error(`No resume found with id: ${reviewId}`);
```
Comment (markyd-XXI) [inline app/controllers/review.controller.js]:
```suggestion
            console.error(`No review found with id: ${reviewId}`);
```

File: app/routes/review.routes.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ module.exports = (app) => {
     router.put("/review/:id", [authenticateRoute], Review.update);
   
     //Delete a Reviews for resume
-    router.delete("/review/:id", [authenticateRoute], Review.delete);
+    router.delete("/reviews/:id", Review.delete);
```
Comment (mmendiolajr) [inline app/routes/review.routes.js]: Why did you delete the [authenticateRoute]?

File: app/controllers/review.controller.js [hunk 3]
```diff
@@ -44,11 +44,39 @@ exports.create = async (req, res) => {
...
+            return res.status(500).send({
+                message: 'No resume found',
```
Comment (markyd-XXI) [inline app/controllers/review.controller.js]:
```suggestion
                message: 'No review found',
```

File: app/controllers/review.controller.js [hunk 4]
```diff
@@ -44,11 +44,39 @@ exports.create = async (req, res) => {
...
+            });
+        }
+
+        try {
+            return res.send({ message: 'Resume deleted successfully', success: true });
+        } catch (error) {
+            console.error(`An error occurred while sending the PDF response: ${error.message}`);
+            return res.status(500).send({
+                message: "An error occurred while returning the PDF.",
```
Comment (mmendiolajr) [inline app/controllers/review.controller.js]: Returning PDF, did you leave this in here by accident?

**Review by mmendiolajr — APPROVED** [review]

File: app/controllers/review.controller.js [hunk 5]
```diff
@@ -44,11 +44,39 @@ exports.create = async (req, res) => {
...
+
+        if (result === 0) {
+            console.error(`No resume found with id: ${reviewId}`);
+            return res.status(500).send({
+                message: 'No resume found',
+            });
+        }
+
+        try {
+            return res.send({ message: 'Resume deleted successfully', success: true });
```
Comment (markyd-XXI) [inline app/controllers/review.controller.js]:
```suggestion
            return res.send({ message: 'Review deleted successfully', success: true });
```

File: app/controllers/review.controller.js [hunk 6]
```diff
@@ -44,11 +44,39 @@ exports.create = async (req, res) => {
...
+        } catch (error) {
+            console.error(`An error occurred while sending the PDF response: ${error.message}`);
```
Comment (markyd-XXI) [inline app/controllers/review.controller.js]:
```suggestion
            console.error(`An error occurred while sending the review delete response: ${error.message}`);
```

File: app/controllers/review.controller.js (hunk 4 above)
Comment (markyd-XXI) [inline app/controllers/review.controller.js]:
```suggestion
                message: "An error occurred while deleting the review.",
```

File: app/controllers/review.controller.js [hunk 7]
```diff
@@ -44,11 +44,39 @@ exports.create = async (req, res) => {
...
+            return res.status(500).send({
+                message: "An error occurred while returning the PDF.",
+            });
+        }
+    } catch (error) {
+        console.error("Error deleting resume: ", error);
```
Comment (markyd-XXI) [inline app/controllers/review.controller.js]:
```suggestion
        console.error("Error deleting review: ", error);
```

File: app/controllers/review.controller.js [hunk 8]
```diff
@@ -44,11 +44,39 @@ exports.create = async (req, res) => {
...
+        return res.status(500).send({
+            message: "An error occurred while deleting the resume."
```
Comment (markyd-XXI) [inline app/controllers/review.controller.js]:
```suggestion
            message: "An error occurred while deleting the review."
```

File: app/routes/review.routes.js (hunk 2 above)
Comment (markyd-XXI) [inline app/routes/review.routes.js]: The authenticate route should stay in there.

**Review by markyd-XXI — CHANGES_REQUESTED** [review]
Comments added

**Review by markyd-XXI — APPROVED** [review]
